package-lock.json

```diff
@@ -5,7 +5,6 @@
   "requires": true,
   "packages": {
     "": {
-      "name": "arcsjs-core",
```

Comment:
i messed up, i don't know why this change was made, but also in the spirit of my today's git adventure, i cannot find a way to revert it... i'll keep trying.

Reply:
You can ignore changes to package-lock.json, commit away